timelock-server/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosTimeLockServer.java

```diff
@@ -270,10 +301,7 @@ private ManagedTimestampService createPaxosBackedTimestampService(String client)
 
         PaxosSynchronizer.synchronizeLearner(ourLearner, learners);
 
-        return AwaitingLeadershipProxy.newProxyInstance(
-                ManagedTimestampService.class,
-                () -> createManagedPaxosTimestampService(proposer, client, acceptors, learners),
-                leaderElectionService);
+        return () -> createManagedPaxosTimestampService(proposer, client, acceptors, learners);
     }
```

Comment:
Discussed offline - I was a bit concerned about whether this was directly exposed, but we're clearly good, and I forgot we wrote tests for this (`IsolatedPaxosTimeLockServerIntegrationTest#cannotPerformTimestampManagementWithoutQuorum` explicitly goes to the timestamp management service).